src/IceNucleation.jl

```diff
@@ -54,4 +55,39 @@ function dust_activated_number_fraction(
     end
 end
 
+"""
+   ABIFM_J(prs, x, T)
+
+ - `prs` - set with model parameters
+ - `x` - wt% sulphuric acid / 100
```

Comment:
Could you add some more explanation on what this variable is and what are the units?